### PR TITLE
[TAS-761] ✨ Add about NFT book page

### PR DESCRIPTION
**Pull request by nwingt**
localhost_3000_zh-Hant_about_nft-book
localhost_3000_zh-Hant_about_nft-book_mobile

### CODE REVIEW

File: src/locales/zh-Hant.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 {
+  "about_nft_book_section_1_description": "未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。",
+  "about_nft_book_section_1_title": "讀書人的事，能說偷麼",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:3]:
```suggestion
  "about_nft_book_section_1_title": "讀書人的事，能算偷麼",

```

File: src/locales/en.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
 {
+  "about_nft_book_section_1_description": "未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。",
+  "about_nft_book_section_1_title": "讀書人的事，能說偷麼",
```
Comment (williamchong) [inline src/locales/en.json:3]:
```suggestion
  "about_nft_book_section_1_title": "讀書人的事，能算偷麼",

```

**Review by williamchong — COMMENTED** [review]
Waiting for actual text
Need sections for NFT book functionality features, ownership etc

File: src/pages/about/nft-book.vue [hunk 3]
```diff
@@ -0,0 +1,978 @@
...
+      books.sort((a, b) => {
+        const aMatchedLocale = locale.includes(a.locale);
+        const bMatchedLocale = locale.includes(b.locale);
+        if (aMatchedLocale && !bMatchedLocale) {
+          return -1;
+        }
+        if (!aMatchedLocale && bMatchedLocale) {
+          return 1;
+        }
+        return 0;
```
Comment (williamchong) [inline src/pages/about/nft-book.vue:684]: Did we change this part of logic in another part of code in another PR?

**Review by williamchong — CHANGES_REQUESTED** [review]
Also please update feature text from slack

File: src/pages/about/nft-book.vue (hunk 3 above)
Comment (nwingt) [reply src/pages/about/nft-book.vue:684]: Yes, this should be merged first

File: src/locales/en.json [hunk 4]
```diff
@@ -1,4 +1,25 @@
 {
+  "about_nft_book_section_1_description": "未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。",
```
Comment (williamchong) [inline src/locales/en.json:2]: Remove/edit this?

File: src/locales/zh-Hant.json [hunk 5]
```diff
@@ -1,4 +1,25 @@
 {
+  "about_nft_book_section_1_description": "未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。未來就在你手，電子書你一定要有。買了我們這種才是真正擁有。絕版好書有得救，新舊藏書睇到夠。",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:2]: Remove/edit this?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/locales/zh-Hant.json (hunk 5 above)
Comment (nwingt) [reply src/locales/zh-Hant.json]: Remove first

File: src/locales/en.json (hunk 4 above)
Comment (nwingt) [reply src/locales/en.json]: Remove first

**Review by williamchong — APPROVED** [review]

**Pull request merged**